src/components/link.js

```diff
@@ -49,6 +49,11 @@ export default {
         // don't redirect on right click
         /* istanbul ignore if */
         if (e.button !== 0) return
+        // don't redirect if `target="_blank"`
+        /* istanbul ignore if */
+        const target = this.$el && this.$el.getAttribute('target')
```

Comment:
We should use `e.target` here. `this.$el` may be a wrapping `<li>` if `tag` prop is used.